bool middle_led_ring_button_pressed
float32 left_stick_x
float32 left_stick_y
float32 right_stick_x
float32 right_stick_y
bool left_stick_button_pressed
bool right_stick_button_pressed
bool bottom_button_pressed
bool top_button_pressed
bool left_button_pressed
bool right_button_pressed
bool left_shoulder_button_pressed
bool right_shoulder_button_pressed
bool select_button_pressed
bool start_button_pressed
float32 left_trigger_pulled
float32 right_trigger_pulled
bool bottom_pad_pressed
bool top_pad_pressed
bool left_pad_pressed
bool right_pad_pressed